app/PiCalculator/pi.js

```diff
@@ -0,0 +1,222 @@
+var numOfPointsToTryPerClient = 100000000000;
+var batchSizePerUpdate = 5000000;
+
+PIData = new Mongo.Collection('pidata');
+var n = 5;
+
+var signedInWithFacebook = false;
+var signedInWithGoogle = false;
+var signedInWithLocalAccount = false;
```

Comment:
AFAIK these values (`signedInWith`[`Facebook`|`Google`|`LocalAccount`]) are no longer relevant after a41ab92.  If so, they should be removed